### ISSUE TITLE
Feasibility of unit testing

### ISSUE THREAD

**Issue by GammaGames** [afranka70/AbOutIssues#25]
# AbOut Version
326s17
# Submitted by
Josh Lee
# Description
Research should be done into possible testing frame works for PHP and the feasibility of unit testing web based services.

**Comment by LoganSWarner** [comment]
Note that this is in progress, and nearly complete.

**Comment by LoganSWarner** [comment]
Jesse has already done part of this, going over the various \_\_Unit frameworks. We need to go further and get something like PHPUnit working.
Major refactoring of the PHP is necessary, and I believe tests will help a lot for incrementally understanding and documenting what the code does and should do (more than Algorithm labels). Tests need to be in place before the end of the semester to really prepare whoever works on this next year.
